OutOfSchool/OutOfSchool.BusinessLogic/Util/MappingProfile.cs

```diff
@@ -489,6 +517,20 @@ public MappingProfile()
             .ForMember(dest => dest.EmailConfirmed, opt => opt.MapFrom(src => src.User.EmailConfirmed))
             .ForMember(dest => dest.UserName, opt => opt.MapFrom(src => src.User.UserName));
 
+        CreateMap<Parent, BaseUpdateUserDto>()
```

Comment:
this mapping should not be possible with new concept of different presentation and update DTOs